tests/testsuite.cpp

```diff
@@ -790,6 +797,88 @@ void test_duration()
     dummy_function();
 }
 
+// boost::multi_index
+
+namespace mi = boost::multi_index;
+
+struct mi_tag_sequenced {};
+struct mi_tag_ordered {};
+struct mi_tag_hashed {};
+
+typedef mi::multi_index_container<
```

Comment:
To be consistent with the rest of the file, c++11-style type aliases should be used, i.e. `using sequenced_first = mi::multi_index_container<...`